src/vmm/src/lib.rs

```diff
@@ -41,7 +41,9 @@ use devices::SerialWrapper;
 
 mod vcpu;
 mod vm;
-use vcpu::{cpuid, mpspec, mptable, Vcpu};
+use crate::vcpu::cpuid::filter_cpuid;
+use crate::vcpu::VcpuState;
```

Comment:
Is `crate::` necessary?

Reply:
I have no idea why, but in the vm-vcpu crate, if I do not write `crate::` it cannot find the vcpu module :(. I left it as is.